playbooks/roles/kibana/templates/nginx.conf.j2

```diff
@@ -0,0 +1,47 @@
+upstream elasticsearch_server {
+   server 127.0.0.1:9200;
+}
+
+server {
+  # Kibana server, templated by ansible
```

Comment:
Since this file:
- has a template file extension
- is located in the Ansible Roles directory

this comment is obvious enough that we can safely drop it.